src/Compilers/VisualBasic/Test/Semantic/Compilation/SemanticModelAPITests.vb

```diff
@@ -4274,5 +4274,31 @@ End Class
         End Sub
 #End Region
 
+        <Fact, WorkItem(1146124, "DevDiv")>
+        Public Sub GetTypeInfoForXmlStringInCref()
+            Dim compilation = CompilationUtils.CreateCompilationWithMscorlibAndVBRuntime(
+<compilation name="GetSemanticInfo">
+    <file name="a.vb"><![CDATA[
+Module Program
+    ''' <summary>
+    ''' <see cref=""/>
+    ''' </summary>
+    ''' <param name="args"></param>
+    Sub Main(args As String())
+
+    End Sub
+End Module
+    ]]></file>
+</compilation>)
+
+            Dim tree As SyntaxTree = (From t In compilation.SyntaxTrees Where t.FilePath = "a.vb").Single()
+            Dim root = tree.GetCompilationUnitRoot
+            Dim xmlString = root.DescendantNodes(descendIntoTrivia:=True).OfType(Of XmlStringSyntax).Single()
```

Comment:
It may be formally unambiguous, but not clear to the reader why `""` is an XmlStringSyntax and `"args"` is not. The simplest thing to make it more clear would be to delete the `param` tag, but even more clear would be to select the node by its contents.